content/content-tool/tool/src/java/org/sakaiproject/content/entityproviders/ContentEntityProvider.java

```diff
@@ -161,7 +161,18 @@ public boolean entityExists(String id) {
 	public List<EntityContent> getResources(EntityView view, Map<String, Object> params)
 			throws EntityPermissionException {
 		String userId = developerHelperService.getCurrentUserId();
-		if (userId == null) {
+		boolean isPublicSite = false;
```

Comment:
Don't the underlying `ContentHostingService` methods do a permission check? So do you need this additional check at all?